src/mvs/mvsNodeActions.ts

```diff
@@ -33,9 +33,19 @@ export async function uploadDialog(node: ZoweNode, datasetProvider: DatasetTree)
     datasetProvider.refresh();
 }
 
+export function getDatasetLabel(node: ZoweNode) {
+    if (node.mParent && node.mParent.contextValue === "favorite") {
+        const profileEnd = "]: ";
```

Comment:
We do need to refactor this code at some point to store dataset name and profile name as separate attributes on the node object instead of concatenating the strings.